charts/opentelemetry-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.40.0
+version: 0.41.0
```

Comment:
Please change this to be 0.40.1 instead of 0.41.0. This change only needs the patch number to be changed. Also please run "make generate-examples".